bgpd/bgp_labelpool.c

```diff
@@ -937,30 +1051,508 @@ DEFUN(show_bgp_labelpool_chunks, show_bgp_labelpool_chunks_cmd,
 		}
 		json = json_object_new_array();
 	} else {
-		vty_out(vty, "First    Last\n");
-		vty_out(vty, "--------------\n");
+		vty_out(vty, "%10s %10s %10s %10s\n", "First", "Last", "Size",
+			"nfree");
+		vty_out(vty, "-------------------------------------------\n");
 	}
 
 	for (ALL_LIST_ELEMENTS_RO(lp->chunks, node, chunk)) {
+		uint32_t size;
+
+		size = chunk->last - chunk->first + 1;
+
 		if (uj) {
 			json_elem = json_object_new_object();
 			json_object_array_add(json, json_elem);
 			json_object_int_add(json_elem, "first", chunk->first);
 			json_object_int_add(json_elem, "last", chunk->last);
+			json_object_int_add(json_elem, "size", size);
+			json_object_int_add(json_elem, "nfree", chunk->nfree);
```

Comment:
numberFree

Reply:
Do you mean the structure field name or the json name?

Reply:
Based on json names in ospf, I guessed you meant the json name. Fixed.